### PR TITLE
Documentate limitations on WSL

### PR DESCRIPTION
**Pull request by LodewijkSioen**
See #31

### CODE REVIEW

**Comment by grahamegrieve** [comment]
The documentation should also mention that you can just run the IG publisher directly in windows if the docker is slow

File: images/ig-publisher-base/README.md [hunk 1]
```diff
@@ -92,3 +92,29 @@ resources in your host filesystem, you can mount them from elsewhere. For
 instance, to bring in the `.fhir` cache from your host machine home directory:
 
     -v /home/$USER/.fhir:/home/publisher/.fhir
+
+## Running on WSL2
+Using this image to build your IG using Docker on WSL2 is really slow due to a 
+known limitation when mounting the Windows filesystem. This can be circumvented
```
Comment (jmandel) [inline images/ig-publisher-base/README.md:98]:
```suggestion
known limitation when mounting the Windows filesystem. You can run the IG publisher without docker by following the instructions at https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation#IGPublisherDocumentation-Installing; or you can improve performance in WSL2
```

Comment (jmandel) [reply images/ig-publisher-base/README.md:98]: Added a proposed edit to the PR, to address Grahame's point; see if you're okay with it @LodewijkSioen ?

Comment (LodewijkSioen) [reply images/ig-publisher-base/README.md:98]: Okay for me. But I think that people looking at this docker image will already know this. We want to use docker as a way to minimize the number of dependencies people have to install on their PC when working on IGs.

**Pull request merged**